scripts/install-cni.sh

```diff
@@ -14,21 +14,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+log() {
+  echo "[${EPOCHREALTIME}]" "$@"
```

Comment:
Users will ingest these logs via stackdriver or other logging system -- can we rely on those to provide the timestamps from the container logs?

Reply:
I made them for easier observation from direct `kubectl logs` calls. Things like kubelet logs also have timestamps.

Reply:
If it's a `$(date)` call I wouldn't do it, but `${EPOCHREALTIME}` being bash builtin I assume the overhead is negligible?

Reply:
Also in case of ingestion delay - we're trying to optimize for sub-second delay here.

Reply:
The container logs already have timestamps:

```
$ kubectl logs example --timestamps
2024-05-21T18:44:08.265571760Z Hello, World!
```

This is the same timestamp provided by stackdriver in the `timestamp` field (and is distinct from `receivedTimestamp`).

Reply:
Wow didn't know about this, but let's leave the `log` call for further extension (e.g. some messages to be written to stderr).

Reply:
SGTM